grunttasks/xssmitigate.js

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var xss = require('xss');
+
+module.exports = function (grunt) {
+  grunt.registerTask('xss-flag', 'Flag potentially harmful files', [
```

Comment:
``` 'Flag potentially harmful files'``` need a better description + a comment to explain what this task does.
If you read without context then `Flag potentially harmful files` does not explain what `files` and what exactly is it looking for, what is `harmful`?